src/views/Playground.vue

```diff
@@ -73,8 +75,8 @@ const scoresWithZeroBalanceAddresses = computed(() => {
     return null;
   }
   // If an address is not present inside the scoresObject, add it with a zero balance
-  const addressesArray = form.value.addresses ?? [];
-  const scoresObject = scores.value[0] ?? {};
+  const addressesArray = (form.value.addresses ?? []).map(getAddress);
+  const scoresObject = mapKeys(getAddress, scores.value[0] ?? {});
```

Comment:
Shouldn't the the Object be the first param and the function be the second? https://www.geeksforgeeks.org/lodash-_-mapkeys-method/

Reply:
Hey @samuveth this is lodash's FP module function
https://github.com/lodash/lodash/wiki/FP-Guide

It is same as normal but iteratee-first data-last methods and easy to read 😅 

Else normal lodash function will be like below (key is in second param):
```js
mapKeys(scores.value[0] ?? {}, (value, key) => getAddress(key))
```

Reply:
But this is not required, should i use to normal lodash?

Reply:
Got it :)